src/libraries/Microsoft.Extensions.DependencyInjection/src/DependencyInjectionEventSource.cs

```diff
@@ -14,9 +16,16 @@ internal sealed class DependencyInjectionEventSource : EventSource
     {
         public static readonly DependencyInjectionEventSource Log = new DependencyInjectionEventSource();
 
-        // Event source doesn't support large payloads so we chunk formatted call site tree
+        public static class Keywords
+        {
+            public const EventKeywords ServiceProviderInitialized = (EventKeywords)0x1;
```

Comment:
I think you might need to specify a keyword for the other events in this provider now.  If you turn on keyword 0x1, you'll just get the ServiceProviderInitialized events.  If you specify keyword 0, I'm not sure you'll get the others - worth checking via ETW.

Reply:
>  If you turn on keyword 0x1, you'll just get the ServiceProviderInitialized events.

That was my intention. The existing events all check for `EventKeywords.All`:

https://github.com/dotnet/runtime/blob/a1aeb132ca60becb1e8ffe02150d59782c1214c2/src/libraries/Microsoft.Extensions.DependencyInjection/src/DependencyInjectionEventSource.cs#L80-L82

So if you don't specify `All`, you won't get those events.

> If you specify keyword 0, I'm not sure you'll get the others - worth checking via ETW.

When I do

```
dotnet trace collect --providers Microsoft-Extensions-DependencyInjection:0:Verbose --name OrchardCoreTest
```

I don't get any events logged.

When I do

```
 dotnet trace collect --providers Microsoft-Extensions-DependencyInjection::Verbose --name OrchardCoreTest
```

I get them all.

And when I do

```
dotnet trace collect --providers Microsoft-Extensions-DependencyInjection:1:Verbose --name OrchardCoreTest
```

I just get the 2 new ones.